src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,54 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { getUser } from '../../api/api';
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: null,
+  };
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    getUser(this.props.userId)
+      .then(user => this.setState({ user }));
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.userId !== this.props.userId) {
+      getUser(this.props.userId)
+        .then(user => this.setState({ user }));
+    }
+  }
+
+  render() {
+    const { clearUser } = this.props;
+    const { user } = this.state;
+
```

Comment:
before return statement you can check if user exists, and return null otherwise. Then you won't need to check if the user is found 4 times